src/mapping/PartitionOfUnityMapping.hpp

```diff
@@ -232,7 +232,10 @@ void PartitionOfUnityMapping<RADIAL_BASIS_FUNCTION_T>::computeMapping()
     const auto localNumberOfClusters = clusterIDs.size();
     // Consider the case where we didn't find any cluster (meshes don't match very well)
     if (localNumberOfClusters == 0) {
-      PRECICE_ERROR("Output vertex {} could not be assigned to any cluster. This means that the meshes probably do not match well geometry-wise or the target number of vertices per cluster is too small.", vertex.getCoords());
+      PRECICE_ERROR("Output vertex {} of mesh \"{}\" could not be assigned to any cluster in the rbf-pum mapping. This means that the meshes probably do not match well geometry-wise."
+                    " If you want to keep your meshes, you can try to increase the number of \"vertices-per-cluster\" (default is 100), the \"relative-overlap\" (default is 0.3)"
+                    " or disable the option \"project-to-input\".",
```

Comment:
If a specific tag is required, let's make that explicit:

```suggestion
                    " or disable the option \"project-to-input\"."
                    "These options are only valid for the <mapping:rbf-pum-direct/> tag.",
```